rocketpy/Rocket.py

```diff
@@ -442,56 +466,35 @@ def addMotor(self, motor, position):
-            Aerodynamic surface to be added to the rocket. See AerodynamicSurface class
-            for more information.
-        position : int, float
-            Position, in m, of the aerodynamic surface's center of pressure relative to
-            the user defined rocket coordinate system. See `Rocket.coordinateSystemOrientation`
-            for more information regarding the rocket's coordinate system.
-
-        Returns
-        -------
-        None
-        """
-        self.aerodynamicSurfaces.append(surface)
-        surface.position = position
-        if isinstance(surface, NoseCone):
-            self.nosecone.append(surface)
-        elif isinstance(surface, (TrapezoidalFins, EllipticalFins)):
-            self.fins.append(surface)
-        elif isinstance(surface, Tail):
-            self.tail.append(surface)
-        self.evaluateStaticMargin()
-        return None
-
     def addSurfaces(self, surfaces, positions):
-        """Adds multiple aerodynamic surfaces to the rocket. The aerodynamic surfaces
-        must be instances of classes that inherit from the AerodynamicSurface
+        """Adds one or more aerodynamic surfaces to the rocket. The aerodynamic
+        surface must be an instance of a class that inherits from the
+        AerodynamicSurface
 
         Parameters
         ----------
-        surfaces : list of AerodynamicSurface, NoseCone, TrapezoidalFins, EllipticalFins, Tail
-            List of aerodynamic surfaces to be added to the rocket. See AerodynamicSurface class
-            for more information.
-        positions : list of int, float
-            List of positions, in m, of the aerodynamic surfaces' center of pressure relative to
-            the user defined rocket coordinate system. See `Rocket.coordinateSystemOrientation`
-            for more information regarding the rocket's coordinate system.
+        surfaces : list, AeroSurfaces, NoseCone, TrapezoidalFins, EllipticalFins, Tail
+            Aerodynamic surface to be added to the rocket. Can be a list of
+            AeroSurfaces if more than one surface is to be added.
+            See AeroSurface class for more information.
+        positions : int, float, list
+            Position, in m, of the aerodynamic surface's center of pressure
+            relative to the user defined rocket coordinate system.
+            See `Rocket.coordinateSystemOrientation` for more information
+            regarding the rocket's coordinate system.
+            If a list is passed, will correspond to the position of each item
+            in the surfaces list.
 
         Returns
         -------
         None
         """
-        for surface, position in zip(surfaces):
-            self.addSurface(surface, position)
-            surface.position = position
+        try:
+            for surface, position in zip(surfaces, positions):
+                self.aerodynamicSurfaces.add(surface, position)
+        except:
```

Comment:
Try to pinpoint the exception type.

Reply:
Addresed in https://github.com/RocketPy-Team/RocketPy/pull/355/commits/1d8cb9cea1f5937cffff382ffcf92b6b52983120